### PR TITLE
bugfix: fix variational dropout and rocket launching

### PR DESCRIPTION
**Pull request by tiankongdeguiji**

### CODE REVIEW

File: easy_rec/python/tools/feature_selection.py [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+      if embedding_wise_variational_dropout:
+        self._visualize_embedding_dim_importance(feature_dim_dropout_p)
+      self._visualize_feature_importance(feature_importance)
+
+  def _feature_dim_dropout_ratio(self):
+    """Get dropout ratio of embedding-wise or feature-wise."""
+    config = config_util.get_configs_from_pipeline_file(self._config_path)
+    embedding_wise_variational_dropout = config.model_config.variational_dropout.embedding_wise_variational_dropout
+    features_dim = {
+        cfg.input_names[0]: cfg.embedding_dim
```
Comment (dawn310826) [inline easy_rec/python/tools/feature_selection.py:77]: raw_feature没有embedding_dim，如何处理

File: easy_rec/python/tools/feature_selection.py [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+    if self._visualize:
+      tf.logging.info('Visualizing feature importance ...')
+      if embedding_wise_variational_dropout:
+        self._visualize_embedding_dim_importance(feature_dim_dropout_p)
+      self._visualize_feature_importance(feature_importance)
+
+  def _feature_dim_dropout_ratio(self):
+    """Get dropout ratio of embedding-wise or feature-wise."""
+    config = config_util.get_configs_from_pipeline_file(self._config_path)
+    embedding_wise_variational_dropout = config.model_config.variational_dropout.embedding_wise_variational_dropout
```
Comment (dawn310826) [inline easy_rec/python/tools/feature_selection.py:75]: assert config.model_config.HasField('variational_dropout'), 'variational_dropout must be in model_config'

File: easy_rec/python/tools/feature_selection.py (hunk 1 above)
Comment (tiankongdeguiji) [reply easy_rec/python/tools/feature_selection.py:79]: done

File: easy_rec/python/tools/feature_selection.py (hunk 2 above)
Comment (tiankongdeguiji) [reply easy_rec/python/tools/feature_selection.py:77]: done

File: easy_rec/python/layers/input_layer.py [hunk 3]
```diff
@@ -233,8 +233,12 @@ def single_call_input_layer(self,
           else:
             raise NotImplementedError
       if self._variational_dropout_config is not None:
```
Comment (chengmengli06) [inline easy_rec/python/layers/input_layer.py:235]: wait for ut pass

**Review by chengmengli06 — CHANGES_REQUESTED** [review]

**Review by chengmengli06 — APPROVED** [review]

**Pull request merged**